builder/dimension/lookup.go

```diff
@@ -32,6 +32,11 @@ func (l *Lookup) SetName(name string) *Lookup {
 	return l
 }
 
+func (l *Lookup) SetOutputName(outputName string) *Lookup {
+	l.OutputName = outputName
```

Comment:
Probably a good idea to call the base setter instead.
```suggestion
	l.Base.SetOutputName(outputName)
```

Reply:
Good catch. Just corrected it.